lib/runner/cli/cli.js

```diff
@@ -128,9 +128,27 @@ class CliRunner {
         '  Please consult the docs at: https://nightwatchjs.org/gettingstarted#settings-file');
     }
 
+    this.validateChromeOptions();
+
     return this;
   }
 
+  validateChromeOptions() {
+    Object.keys(this.baseSettings.test_settings).map((testSettingKey) => {
+      let testSetting = this.baseSettings.test_settings[testSettingKey];
+      if (Utils.isObject(testSetting.desiredCapabilities) &&
+        Utils.isObject(testSetting.desiredCapabilities.chromeOptions) &&
+        Utils.isObject(testSetting.desiredCapabilities.chromeOptions.args)) {
+        this.invalidChromeArguments = Utils.findUnquotedCliArgumentsWithWhitespaces(testSetting.desiredCapabilities.chromeOptions.args);
+
+        if(this.invalidChromeArguments && this.invalidChromeArguments.length > 0){
+          console.error('Unquoted argument value contains whitespace which may leads to errors : [ ' + this.invalidChromeArguments +
```

Comment:
I still think you want this to be "lead", not "leads".

Reply:
done :)